e2e-tests/portal/cypress/e2e/journeys/dashboard/dashboard-deals/dashboard-deals-filters.spec.js

```diff
@@ -15,7 +15,7 @@ context('Dashboard Deals filters', () => {
     cy.deleteGefApplications(ADMIN);
     cy.deleteDeals(ADMIN);
 
-    cy.insertOneDeal(BSS_DEAL_DRAFT, BANK1_MAKER1).then((deal) => {
+    cy.createBssDeal({ readyForCheck: true }).then((deal) => {
```

Comment:
This should be `cy.createBssDeal({});` so that the deal has a `DRAFT` status like the previous test